src/pages/index.js

```diff
@@ -1,16 +1,75 @@
 import React from "react"
-
 import Layout from "../components/layout"
 import SEO from "../components/seo"
 import "../styles/index.scss"
+import {graphql} from "gatsby"
+
+
+class IndexPage extends React.Component{
+    render(){
+        const {data} = this.props;
+        const modules = data.allContentfulModuleTextandCta.edges;
+        const heroModule =data.allContentfulHeroModule.edges;
+      return (
+        <Layout>
+          <SEO title="Home" />
+            {heroModule.map((module)=>{
+                return (
+                    <div className={`hero__container`}>
+                        <img className={`hero__image`} src={`${module.node.heroImage.file.url}`} alt={``}/>
+                        <h1 className={`hero__title`}>{module.node.heroHeading}</h1>
+                    </div>
+                )
+            })}
 
-const IndexPage = () => (
-  <Layout>
-    <SEO title="Home" />
-    <h2>Hi people</h2>
-    <p>Welcome to Equal Experts Germany</p>
-    <p>Let's build something great.</p>
-  </Layout>
-)
+            {
+                modules.map((module) => {
+                    return(
+                        <div className={`module`}>
+                            <div className={`module__title`}>{module.node.moduleTitle}</div>
+                            <div className={`module__content`} dangerouslySetInnerHTML={{__html: module.node.moduleContent.childContentfulRichText.html}} />
+                            <div className={`module__link`} dangerouslySetInnerHTML={{__html: module.node.moduleCtAlink.childContentfulRichText.html}} />
```

Comment:
what if there is no CTA link? I believe it's optional. so it would be better to only render this if it is present, than an empty div.

Reply:
updated to not show empty div.